terraform/tooling_invoker/launch_template.tf

```diff
@@ -1,6 +1,6 @@
 resource "aws_launch_template" "main" {
   image_id                = "ami-03c207961a2d90a8e"
-  instance_type           = "t3.small"
+  instance_type           = "t4g.small"
```

Comment:
Am I correct in reading that this new instance type is ARM-based?

Reply:
Ah. This is why I didn't do this before...! I presume we can't use ARM easily?

Reply:
Correct. A lot of the Docker images won't be multi-platform